google_suggest/__init__.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+
+"""Get Google suggestions"""
+from albertv0 import *
+from locale import getlocale
+from urllib import request, parse
+import xml.etree.ElementTree as etree
+import os
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Google Suggest"
+__version__ = "0.1"
+__trigger__ = "g "
+__author__ = "Hans C. Jørgensen"
+__dependencies__ = []
+
+iconPath = iconLookup('google_suggest')
+if not iconPath:
+    iconPath = os.path.dirname(__file__)+"/google_suggest.svg"
+google_search_baseurl = 'http://www.google.com/search'
+google_suggest_baseurl = 'http://google.com/complete/search'
+user_agent = "Mozilla/5.0"
+limit = 20
+
+
+def initialize():
+    global baseurl
+
+def handleQuery(query):
+    if query.isTriggered:
+
```

Comment:
I would consider adding:
sleep(0.2)
if not query.isValid:
            return

That way avoid being rate-limited by google.
I've tried it with a couple of other google end points, and after a while they always end up limit the number of calls you can make to their API endpoint.

Reply:
I wasn't aware that they rate-limited. Thanks for the heads up.
I'll put it in.
Will also go see if I can find specifics on how the rate limiting functions, so I can add a warning in a comment for anybody who wants to alter the code.

Reply:
I added your suggested change. 

Unfortunately I was unable to find any definite numbers on when and how the rang-limiting kicks in, so I left it out.